AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Utils/Version+Utils.swift

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import ArgumentParser
+import PackageDescription
+
+// MARK: - Decodable
+
+extension Version: Decodable {
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        let value = try container.decode(String.self)
+        self.init(stringLiteral: value)
+    }
+}
+
+// MARK: - Loading from File
+
+extension Version {
+    /// Returns a version loaded from the provided file.
+    /// The file's contents must only contain the version and nothing else.
+    /// This is used to load a Version from a `Package.version` file.
+    ///
+    /// - Parameter filePath: The path to file containing the version
+    /// - Returns: A version loaded from the provided file.
+    static func fromFile(_ filePath: String) throws -> Version {
```

Comment:
a command on the format of the file would makes it clearer.

Also, possibly tests.